crates/next-dev-tests/tests/integration.rs

```diff
@@ -493,15 +492,13 @@ impl IssueReporter for TestIssueReporter {
         _source: TransientValue<RawVc>,
     ) -> Result<BoolVc> {
         let issue_tx = self.issue_tx.get_untracked().clone();
-        let mut has_fatal = false;
         for issue in captured_issues.iter() {
-            let plain = issue.into_plain().await?;
-            if plain.severity == IssueSeverity::Fatal {
-                has_fatal = true;
-            }
-            issue_tx.send(plain).await?;
+            let plain = issue.into_plain();
+            issue_tx
+                .send((plain.await?, (*(plain.dbg().await?)).clone()))
```

Comment:
Same `clone_value` change. We could also make the parameter a `ValueDebugStringReadRef` and not need to clone the string.